src/res/translation/translation_nl_NL.ts

```diff
@@ -3929,7 +3929,7 @@ We hebben uw kanaal gedempt en &apos;Demp mijzelf&apos; geactiveerd. Los eerst h
         <translation>&amp;Window</translation>
     </message>
     <message>
-        <location filename="../../util.h" line="610"/>
+        <location filename="../../util.h" line="606"/>
```

Comment:
I don't think it's relevant, but I'm not sure why this happened. Have you been on `master` when performing the translations?

Reply:
I was on master (which I updated first to be in sync) before adding the translation. Though I did pre-work on the translation and did copy text from another file which I tested first.
I will update the PR and correct this line (and line 2097).

Reply:
>  I will update the PR and correct this line (and line 2097).

I don't think it's relevant, probably more a pointer. I was just wondering how this could happen.

Reply:
Resolved it already, it should not have happened. Well spotted!